lib/wrappers/rust/Readme.md

```diff
@@ -0,0 +1,17 @@
+# Rust wrapper 
+
+The rust wrapper library wrapping the cw-core library, which is used to control the contour wall. The examples of using rust wrapper is put at the end of the lib.rs file.
+
+# How to run
+1. Install Cargo Rust version >=  1.75
+```
+[lib]
+crate-type = ["dylib"]
+```
```

Comment:
You just have a code block here, without any text around it.

Reply:
done!